packages/core/datadog-configuration.gradle

```diff
@@ -0,0 +1,23 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+def getConfigurationFilePath() {
+    if (project.hasProperty("datadog")) {
+        if (project.datadog.configurationFilePath) {
+            return project.datadog.configurationFilePath
+        }
+    }
+    return '../../datadog-configuration.json'
+}
+
+task copyDatadogConfigurationFile(type: Copy) {
+    from getConfigurationFilePath()
+    into 'src/main/assets'
```

Comment:
it is better to copy things somewhere inside the `build` folder, this way you won't need to add file to `.gitignore` manually.

Check this out, for example https://github.com/DataDog/dd-sdk-android-gradle-plugin/blob/92de1348d4d39351e0192c388ea564305de6dd8f/dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/DdAndroidGradlePlugin.kt#L156 (but this may require also to add some task dependencies).